modules/ocha_ai_job_tag/ocha_ai_job_tag.module

```diff
@@ -71,6 +46,11 @@ function ocha_ai_job_tag_node_update(EntityInterface $node) {
     return;
   }
 
+  \Drupal::logger('debug')->notice('status: ' . $node->moderation_status->value);
```

Comment:
this implies moderation / workflow is also a requirement.

Reply:
The `ocha_ai_job_tag` module makes the assumption of working on ReliefWeb job nodes for which the `moderation_status` field is a mandatory base field so in theory always present. Pinging @attiks for awareness of your reservation.